libportal/portal.c

```diff
@@ -32,6 +32,20 @@
 #endif
 #include <stdio.h>
 
+const gchar *
+portal_get_bus_name(void)
```

Comment:
Style nit: missing space before `(`

Reply:
sigh, switching between gnome and not-gnome is *hard*...


All fixed, thanks for the review!